### PR TITLE
10d how to use kv store 

### PR DESCRIPTION
**Pull request by JamieDawson**
This document is for `10.d how to use kv store`.

I added info on all the sections except for `Advanced reference (redis sdk)` because I already go over stuff like that in `10b. two types of integrated storage - kv store` .

Image:
10d

### CODE REVIEW

File: docs/kv-store-how-to-use-it.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+There are several commands you can run with the CLI to interact with your Key-Value Storage. You can see the entire list of commands by running `nim kv list` in the commands prompt.
```
Comment (rabbah) [inline docs/kv-store-how-to-use-it.md:1]: `nim kv` lists the commands. `nim kv list` lists the keys.

File: docs/kv-store-how-to-use-it.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+In this document, we're going over a few commands you can use.
+
+### get list
```
Comment (rabbah) [inline docs/kv-store-how-to-use-it.md:5]: uses more descriptive headings as in "list all keys", "set a key", etc.

File: docs/stateful-api-examples.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Stateful API examples
```
Comment (rabbah) [inline docs/stateful-api-examples.md:1]: does this overlap with your other pr #23?

**Review by rabbah — APPROVED** [review]

File: docs/stateful-api-examples.md (hunk 3 above)
Comment (JamieDawson) [reply docs/stateful-api-examples.md:1]: I personally think good to separate them like this. That file is centered around understanding the concepts and using them in your code. Whereas this file (#24) is more centered around the CLI.

File: docs/stateful-api-examples.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+![](https://live.staticflickr.com/65535/50654177956_ea0933b3d4_k.jpg)
+
+### Using Redis
+
+Redis is a Key-Value store used to keep track of the session state. It comes natively with Nimbella, so you don't have to download and configure Redis into your application.
+
+A Key-Value store acts as a database that you can call on. You can create keys and apply values to them.
+
+![](https://live.staticflickr.com/65535/50653417258_e27306c044_k.jpg)
```
Comment (rabbah) [inline docs/stateful-api-examples.md:21]: can you import/add the images here instead of flickr?

File: docs/stateful-api-examples.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+# Example: Using Redis to track page visits
+
+Let's say you want to keep track of how many people visit your application. Doing that is as simple as calling on Redis from Nimbella in your serverless function, creating a variable to keep track of the count, telling Redis to update the keys value, and returning it.
+
+Here is an example of the same serverless function that calls on Redis written in three different languages.
+
+### JavaScript
```
Comment (rabbah) [inline docs/stateful-api-examples.md:29]: can you apply the multi-tab pattern?

**Comment by JamieDawson** [comment]
There are several issues with this document that I wasn't aware of. Updating it now!

**Comment by JamieDawson** [comment]
1) Removed `stateful-api-examples.md` since that was suppose to be in [#18](https://github.com/nimbella/docs/pull/18). I realized I left the wrong file in for the new branch.
2) Connected the correct file to the sidebar.

Pending re-review on changes.

File: docs/kv-store-how-to-use-it.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+nim kv get [keyName]
+```
+
+### Set a value for a key.
+
+Here's how you can change the value of a key in your command prompt. In this example, we'll be using a key that is holding onto a number called `counter`.
+
+```
+nim kv set counter 42
+>OK
```
Comment (rabbah) [inline docs/kv-store-how-to-use-it.md:35]: please double check the output. I don't think there's a leading `>` here or in the next command.

**Comment by JamieDawson** [comment]
Removed ">" and fixed all conflicts.

**Comment by JamieDawson** [comment]
The file that needs to be checked:
File location:  Developing and Deploying Serverless APIs  ---> Basic KV Commands

**Comment by JamieDawson** [comment]
@rabbah , please confirm the last 4 pull requests I have are complete or not so that I may finalize my other milestones for this week.

**Pull request merged**